internal/app/machined/pkg/controllers/runtime/kernel_param_spec.go

```diff
@@ -95,17 +95,17 @@ func (ctrl *KernelParamSpecController) Run(ctx context.Context, r controller.Run
 
 			for i, item := range list {
 				spec := item.(runtime.KernelParam).TypedSpec()
-				key := item.Metadata().ID()
+				id := item.Metadata().ID()
 
-				if value, duplicate := touchedIDs[key]; i >= configsCounts && duplicate {
-					logger.Warn("overriding KSPP enforced parameter, this is not recommended", zap.String("key", key), zap.String("value", value))
+				if value, duplicate := touchedIDs[spec.Key]; i >= configsCounts && duplicate {
+					logger.Warn("overriding KSPP enforced parameter, this is not recommended", zap.String("key", spec.Key), zap.String("value", value))
```

Comment:
This isn't necessarily KSPP-enforced values, is it?  It's any Talos-asserted default value?

Reply:
no it's actually KSPP ones, not defaults.